webserver.ps1

```diff
@@ -0,0 +1,24 @@
+# Install Chocolatey
+Write-Host "Installing Chocolatey ..." -BackgroundColor DarkMagenta
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+
+# Install VirtualBox with multipass (including extenstions and guest additions)
+Write-Host "Installing Multipass with VirtualBox as hypervisor ..."  -BackgroundColor DarkMagenta
+choco install virtualbox multipass
+choco install virtualbox.extensionpack
```

Comment:
could line 8 and 9 just be rolled into line 7 something like:

`choco install virtualbox multipass virtualbox.extensionpack virtualbox-guest-additions-guest.install`

Reply:
Made required changes